core-services/prow/02_config/_plugins.yaml

```diff
@@ -256,27 +256,6 @@ cherry_pick_unapproved:
   comment: This PR is not for the master branch but does not have the `cherry-pick-approved`  label.
     Adding the `do-not-merge/cherry-pick-not-approved`  label.
 config_updater:
-  cluster_groups:
```

Comment:
why this file has been changed? /hold

Reply:
It was a bug in repo-init (DPTP-2721) that should be fixed now but the PR was likely prepared by a version before the fix

Reply:
The change in this file needs to be removed from the PR @Fedosin

Reply:
@petr-muller @droslean thanks! I removed that generated piece of code.